### ISSUE TITLE
Prevent Firefox from initiating download

### ISSUE THREAD

**Issue by wfaulk** [zaidka/cliget#77]
Some web sites apparently create download-once links, and that means that if Firefox starts downloading the file, trying to use the generated `curl` command doesn't work because the server has detected that a download has already been attempted from elsewhere, despite the fact that the command would likely work if that hadn't happened.

Is it possible to get cliget to prevent Firefox from initiating the download at all?

**Comment by rieje** [comment]
Agreed, also so commands will rename the file automatically unless you delete the one automatically produced by Firefox as soon as it initiated the download.

I think a way to easily implement this would be to have the user copy the link and give it to cliget for it to produce the link so Firefox never attempts to download.

**Comment by Jeronimo17** [comment]
This is key and very important. Many servers only allow one download attempt, and if Firefox uses it, what was generated for curl is no longer valid.
